plugins/woocommerce/templates/cart/cart.php

```diff
@@ -53,7 +54,7 @@
 									sprintf(
 										'<a href="%s" class="remove" aria-label="%s" data-product_id="%s" data-product_sku="%s">&times;</a>',
 										esc_url( wc_get_cart_remove_url( $cart_item_key ) ),
-										esc_html__( 'Remove this item', 'woocommerce' ),
+										sprintf( esc_attr__( 'Remove %s', 'woocommerce' ), $product_name ),
```

Comment:
In this line, only the translated string "Remove %s" is escaped. Not the product name. The product name is filled in _after_ the escape.

I think we should do it like this, to ensure the product name is escaped correctly, too:

```suggestion
										esc_attr( sprintf( __( 'Remove %s', 'woocommerce' ), $product_name ) ),

```